### PR TITLE
Dynamic wiki links

### PR DESCRIPTION
**Pull request by robthomson**
Dynamic links to wiki in configurator.

Links are defined in a javascript file wiki.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,4 @@ nbproject/
 # Eclipse
 .project
 .settings/
+package.json
```
Comment (rotorflight) [inline .gitignore:33]: This is incorrect. package.json is in git

File: src/js/gui.js [hunk 2]
```diff
@@ -398,10 +398,20 @@ GuiControl.prototype.content_ready = function (callback) {
     this.switchery();
 
     if (CONFIGURATOR.connectionValid) {
+
+		//find page title
+        const pageTitle = $('div#content .tab_title');
+		var wikiLoc = pageTitle.attr("i18n");
```
Comment (rotorflight) [inline src/js/gui.js:404]: Please be careful with var. Nearly always you want to use const or let.
I think this should be const.

File: .gitignore (hunk 1 above)
Comment (robthomson) [reply .gitignore:33]: Had to exclude it because my git desktop refused to upload the changes because of validation of existing files failing.

So ignored the file to on theory prevent it being uploaded as I made a local mod to it

File: src/js/gui.js (hunk 2 above)
Comment (robthomson) [reply src/js/gui.js:404]: Surely vat as it changes per page?

File: src/js/wiki.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+wiki["tabAdjustments"] 		= 'https://www.rotorflight.org/docs/Tutorial-Setup/Adjustments';
+wiki["tabSensors"] 			= 'https://www.rotorflight.org/docs/Tutorial-Setup/Sensors';
+wiki["tabBlackbox"] 		= 'https://www.rotorflight.org/docs/Tutorial-Setup/Blackbox';
+wiki["tabCli"] 				= '';
+
+//default wiki url
+const wikiURL =  'https://www.rotorflight.org/';
+
+function buildWikiURL(wikiLoc){
+	if(wikiLoc == '' || wikiLoc == null){
```
Comment (rotorflight) [inline src/js/wiki.js:26]: please use consistent formatting with the rest of the code. This case, the spaces
if (wikiLoc == '' || wikiLoc == null) {

File: package.json [hunk 4]
```diff
@@ -135,7 +135,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint"
+      "pre-commit": ""
```
Comment (rotorflight) [inline package.json:138]: Please ddon't change this in the PR.
It may be a good idea, but it has nothing to do with the wiki pages, so it should go to another PR

Comment (robthomson) [reply package.json:138]: That's why I excluded  the previous file.  Could not work out how to make got commit with an skip on this

File: .gitignore [hunk 5]
```diff
@@ -30,3 +30,4 @@ nbproject/
 # Eclipse
 .project
 .settings/
+
```
Comment (rotorflight) [inline .gitignore:33]: Please remove this change. We don't want to have unnecessary changes in the PRs.

File: src/js/wiki.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+wiki["tabCli"]                 = '';
+
+//default wiki url
+const wikiURL =  'https://www.rotorflight.org/';
+
+function buildWikiURL(wikiLoc){
+    if(wikiLoc == '' || wikiLoc == null){
+            return wikiURL + wikiLoc;
+    } else {
+            loc = wiki[wikiLoc];
```
Comment (rotorflight) [inline src/js/wiki.js:29]: This indentation seems to be wrong

File: src/js/wiki.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+wiki["tabMotors"]             = 'https://www.rotorflight.org/docs/Tutorial-Setup/Motor-and-Esc';
+wiki["tabServos"]             = 'https://www.rotorflight.org/docs/Tutorial-Setup/Servos';
+wiki["tabMixer"]              = 'https://www.rotorflight.org/docs/Tutorial-Setup/Mixer';
+wiki["tabRates"]              = 'https://www.rotorflight.org/docs/Tutorial-Setup/Rates';
+wiki["tabProfiles"]           = 'https://www.rotorflight.org/docs/Tutorial-Setup/Profiles';
+wiki["tabAuxiliary"]          = 'https://www.rotorflight.org/docs/Tutorial-Setup/Modes';
+wiki["tabAdjustments"]        = 'https://www.rotorflight.org/docs/Tutorial-Setup/Adjustments';
+wiki["tabSensors"]            = 'https://www.rotorflight.org/docs/Tutorial-Setup/Sensors';
+wiki["tabBlackbox"]           = 'https://www.rotorflight.org/docs/Tutorial-Setup/Blackbox';
+wiki["tabCli"]                 = '';
```
Comment (rotorflight) [inline src/js/wiki.js:20]: Fix the = to be on the same column

**Review by rotorflight — CHANGES_REQUESTED** [review]
- Please check the indentation. It should be 4 spaces. Now it's randomly 4 or more.
- Please check the formatting with whitespace (to match the rest of the code)

for example not "if(1<2){" but "if (1 < 2) {"

File: src/js/wiki.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+//default wiki url
+const wikiURL =  'https://www.rotorflight.org/';
+
+function buildWikiURL(wikiLoc){
+    if(wikiLoc == '' || wikiLoc == null){
```
Comment (rotorflight) [inline src/js/wiki.js:26]: Whitespace formatting!

Comment (rotorflight) [inline src/js/wiki.js:26]: Whitespaces

File: src/js/wiki.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+            return wikiURL + wikiLoc;
+    } else {
+            loc = wiki[wikiLoc];
+        
+            if(loc == undefined){
```
Comment (rotorflight) [inline src/js/wiki.js:31]: whitespace

File: src/js/wiki.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+                    return wikiURL + wikiLoc;
+            } else if (loc == ''){
```
Comment (rotorflight) [inline src/js/wiki.js:33]: whitespace

File: src/tabs/configuration.html [hunk 11]
```diff
@@ -1,9 +1,9 @@
 <div class="tab-configuration toolbar_fixed_bottom">
     <div class="content_wrapper">
         <div class="tab_title" i18n="tabConfiguration"></div>
-            <div class="cf_doc_version_bt">
-                <a id="button-documentation" href="https://github.com/rotorflight/rotorflight/wiki" target="_blank" rel="noopener noreferrer"></a>
-            </div>
+        <div class="cf_doc_version_bt">
```
Comment (rotorflight) [inline src/tabs/configuration.html:4]: wrong indentation

**Comment by rotorflight** [comment]
merged manually
